COPYING

```diff
@@ -4,67 +4,10 @@ Software in this repositiory is provided under:
 
     SPDX-License-Identifier: GPL-2.0-or-later
 
-(Exception: efitools/VConfig.efi, see below).
-
 NOTE: When this software is run, other software may be
 downloaded, which is provided by the respective authors
 under different open source licenses.
 
 The documentation (README.md) is provided under:
 
     SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
Just curious... where did you get this license identifier?  Do I need to add once of these to my repo?

Reply:
It's from the EDK2 github repo.